plugin-modernizer-core/src/main/java/io/jenkins/tools/pluginmodernizer/core/impl/CacheManager.java

```diff
@@ -1,19 +1,33 @@
 package io.jenkins.tools.pluginmodernizer.core.impl;
 
 import io.jenkins.tools.pluginmodernizer.core.model.ModernizerException;
-import io.jenkins.tools.pluginmodernizer.core.utils.JenkinsPluginInfo;
 import java.io.IOException;
+import java.io.InputStream;
+import java.io.Writer;
+import java.nio.charset.StandardCharsets;
 import java.nio.file.Files;
 import java.nio.file.Path;
+import java.nio.file.attribute.FileTime;
+import java.time.Clock;
+import java.time.Duration;
+import java.util.Scanner;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class CacheManager {
-    private static final Logger LOG = LoggerFactory.getLogger(JenkinsPluginInfo.class);
+    private static final Logger LOG = LoggerFactory.getLogger(CacheManager.class);
     private final Path cache;
```

Comment:
I think this can be renamed `location`to be more clear

```suggestion
    private final Path location;
```